crates/bevy_audio/src/audio.rs

```diff
@@ -188,3 +232,22 @@ where
             .finish()
     }
 }
+
+/// Use this [`Resource`] to control the global volume of all non-absolute audio.
+/// Sources can be set to ignore this setting by setting the `absolute_volume` field on [`PlaybackSettings`] to `true`.
```

Comment:
This sentence is wrong, as that change is done using the `Volume` enum and not a bool.